File: Array/FindAllDuplicatesInAnArray-442/code.cpp
class Solution {
public:
    vector<int> findDuplicates(vector<int>& nums) {
        map<int,int>m;
        vector<int> arr;
        for(int i = 0;i<nums.size();i++){
            m[nums[i]]++;
        }
        int i = 0;
        for(auto n:m){
            if(n.second >= 2){
                arr.push_back(n.first);
                i++;
            }
        }
        return arr;
    }
};

File: Array/SecondLargest-GFG/code.py
class Solution:
    def getSecondLargest(self, arr):
        if len(arr)<2:
            return -1
        largest = secondLargest = -1
        for i in arr:
            if i > largest:
                secondlargest = largest
                largest = i
                
            elif i < largest and i > secondLargest:
                secondLargest = i
                
        return secondLargest